packages/integration-tests/src/test-projects/data-storage-auth-with-triggers-ts/amplify/data/resource.ts

```diff
@@ -26,7 +26,7 @@ const schema = a.schema({
     .returns(a.ref('EchoResponse'))
     .authorization([a.allow.private()])
     .function('echo'),
-});
+}) as never; // Not 100% sure why TS is complaining here. The error I'm getting is "The inferred type of 'schema' references an inaccessible 'unique symbol' type. A type annotation is necessary."
```

Comment:
This PR is now running the test projects through the project tsconfig rather than the tsconfig that is generated for customer projects. I'll do a little more digging to see if there's a quick fix and cut a tech debt issue if not.

Reply:
I'll cut an issue to track this. I'm not finding any useful information on what's going on here but I have confirmed that this schema builds using tsc without issue in a standalone project. When I copy the tsconfig from the standalone project into this directory, it fails with that error message.

Reply:
Created: https://github.com/aws-amplify/amplify-backend/issues/765